### ISSUE TITLE
Commands on another box

### ISSUE THREAD

**Issue by zimonline** [evgeny-golubev/Laravel-4-Artisan#7]
Is there any way this can/could be made to send commands to another box? I develop on a local dev server but more and more people are using Vagrant. It would be nice if it could fire them over ssh.

Many Thanks.

**Comment by evgeny-golubev** [comment]
Yes, it can be added in next update. But i can't do it earlier the end of next week.
